apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -231,6 +231,22 @@ function load_wpcom_block_editor_nux() {
 }
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load_wpcom_block_editor_nux' );
 
+/**
+ * Re-register some core patterns to push them down in the inserter list.
+ * The reason is that Dotcom curate the pattern list based on their look.
+ */
+function re_register_curated_core_patterns() {
```

Comment:
[nit] How about calling it `reorder_curated_core_patterns`?

Reply:
Thanks, I wasn't sure about the name.